eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -97,6 +102,7 @@
                                 <source>..\..\eventhubs\data-plane\azure-eventhubs\src\main\java</source>
                                 <source>..\..\eventhubs\data-plane\azure-eventhubs-eph\src\main\java</source>
                                 <source>..\..\eventhubs\data-plane\azure-eventhubs-extensions\src\main\java</source>
+                                <source>..\..\common\azure-common\src\main\java</source>
                             </sources>
```

Comment:
Are you going to follow-up with azure-common-auth and azure-common-mgmt?

Reply:
Yes. Spotbugs work based on packages.